packages/f-content-cards/src/components/cardTemplates/HomeContentCard.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div
+        :data-test-id="testId"
+        :style="{ background: backgroundColor }"
+        :class="[$style['c-homeContentCard']]">
```

Comment:
Worth adding in a global class here to allow this to be influenced by styles in the consuming application of possible:
```suggestion
        :class="['c-homeContentCard', $style['c-homeContentCard']]">
```

And similarly for other elements using modularized styles below.